cluster/gce/util.sh

```diff
@@ -675,17 +686,40 @@ function kube-up {
   
   create-node-instance-template $template_name
 
+  if [[ ${MAX_INSTANCES_PER_MIG:-500} -le "0" ]]; then
```

Comment:
There's a lot of ${MAX_INSTANCES_PER_MIG:-500} in this check where you compute the instances/mig. How about creating a local var so that you only need to worry about the defaulting once instead of each time.